### ISSUE TITLE
Support for OpenStore to install .apk

### ISSUE THREAD

**Issue by jor1196** [ubports/anbox#12]
as OpenStore you can install the .clicks, it occurred to me when downloading WhatsApp.apk from the official website and click on the file manager to install and it came out as a message that OpenStore could not install it.

**Comment by ITwrx** [comment]
yes, this would be nice to make it easy for normal people to be able install android apps themselves once their nerd family member installs Anbox for them.
